core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/site/edit/SiteEditViewModel.kt

```diff
@@ -32,10 +38,13 @@ data class SiteEditUiState(
         listOf(UstadMobileSystemCommon.UiLanguage("en", "English")),
     val fieldsEnabled: Boolean = true,
     val siteNameError: String? = null,
+    val bottomNavToggleError: String? = null,
     val registrationEnabledError: String? = null,
     val currentSiteTermsLang: UstadMobileSystemCommon.UiLanguage = uiLangs.first(),
+    val permissionLabels: List<Pair<StringResource,Long>> = emptyList()
```

Comment:
These aren't permission labels even though they use the same pattern - call it flagLabels.